src/xdsl/dialects/llvm.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+from dataclasses import dataclass
+
+from xdsl.irdl import ParameterDef, AnyAttr, irdl_op_builder, irdl_attr_definition, AttributeDef, OperandDef, ResultDef, irdl_op_definition
```

Comment:
Can you put parentheses around the imported names? This will allow the formatter to not go over 80 characters!